src/main/java/seedu/address/ui/AssignmentListPanel.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.person.Assignment;
+
+/**
+ * Panel containing the list of persons.
+ */
```

Comment:
Similar to the other comments, the comments use 'persons' instead of 'assignments'